docs/releases/dev.rst

```diff
@@ -21,6 +21,8 @@ Details of changes
 - :djadmin:`refresh_scores` now recalculates user scores and accepts
   multiple usernames.
 - :djadmin:`contributors` has two new options
+- :djadmin:`list_serializers` has been added to view serializers and
```

Comment:
Expanding in a separate section (and linking to it from here) on what serializers are, what they do and whether users need to care about them or not would be highly needed. Otherwise I feel like anyone reading the release notes or documentation will be pretty lost.